descriptors/ModuleDescriptor-template.json

```diff
@@ -346,7 +346,7 @@
     },
     {
       "id": "finance.transactions",
-      "version": "3.0",
+      "version": "4.0",
```

Comment:
please, do not forget to announce major interface update, and update related acq modules